assets/tests/consolidated_1_1_tests.yaml

```diff
@@ -2735,6 +2735,62 @@ tests:
               relation: viewer
             expectation:
               - user:cheetah
+  - name: exclusion_between_userset_and_type
```

Comment:
This test is the same as this test: https://github.com/openfga/openfga/pull/1526/files#diff-9a0d1413173a8311fa9e645a598bc9eb3f23cdc5564254b2c4f67d1c4d568a8bR1336

It was helpful to verify that the change in assertion was confirmed by list objects and check.